### DISCUSSION TITLE
infuse无法播放本地存储的文件

### DISCUSSION THREAD

**Discussion by pzg417** [AlistGo/alist#429] [Q&A] [answered]
添加本地存储后 使用infuse添加webdav服务器

infuse可以正常读取本地存储和阿里云盘的文件目录

也可以正常播放阿里云盘的文件 

但是无法播放本地存储的文件 提示加载错误

尝试一番后发现 infuse可以正常播放 http://ip:5244/d/xxx/xxx.mov 这样路径的本地存储视频

给大佬添麻烦了

**Comment by weizai408** [comment, marked as answer]
这个应该是infuse的问题， 我用infuse挂webdav也提示播放错误，但用filebox这个软件 就没问题。强烈推荐filebox 好用的很 ios平台。

**Reply by pzg417** [reply]
我也有点摸不着头脑 因为使用其他的webdav客户端 比如FE Explorer是没有问题的 

我之所以试了http://ip:5244/d/xxx/xxx.mov 这种链接也是因为发现从网页端选择使用iina播放的话  是通过/d/xxx/xxx这样的路径播放的 而不是/dav/xxx/xxx

**Reply by pzg417** [reply]
刚才下载了您推荐的filebox 感觉真的很赞 期待mac和apple tv版本

**Reply by Dashan-IZ** [reply]
求大佬告知v3版本中的webdav怎么开启啊，配置老是出错，不会用v3版本的webdav了

**Comment by uwang** [comment]
所以结论是换软件？不用 Infuse 么 😢
